### DISCUSSION TITLE
Input password for SSH command

### DISCUSSION THREAD

**Discussion by csgithub2021** [Tyrrrz/CliWrap#181] [Q&A] [answered]
Hi, I'd like to use CliWrap to make SSH connection but I could not find out how to input password after command executed. I immediate get permission denied. Code Ive tried so far:

```
var cmd  = PipeSource.FromString("password") | Cli
.wrap("ssh")
.withArguments("user@host") | (Console.Writeline, Console.Error.Writeline);

await cmd.ExecuteAsync();
```

What should I do to input required password when make SSH connection? I do appreciate any help.

**Comment by Tyrrrz** [comment, marked as answer]
It looks like it's impossible to do directly: https://stackoverflow.com/a/24465355

Maybe you can somehow fool it by wrapping `Cli.Wrap("/bin/sh")` and running it from there?

**Reply by csgithub2021** [reply]
Yes, I also found the same answer. SSH wont accept pseudo terminal for password authentication. I end up using public key and everything is working perfect now. I'd try your suggestion in other time. Thank you so much for great lib. Keep up good work!!!
